catroid/src/main/AndroidManifest.xml

```diff
@@ -91,6 +91,7 @@
         android:label="${appName}"
         android:theme="@style/Catroid"
         android:supportsRtl="true"
+        android:requestLegacyExternalStorage="true"
```

Comment:
to help the reviewer:
have a look at [opt-out-scoped-storage](https://developer.android.com/training/data-storage/use-cases#opt-out-scoped-storage)